workspaces/analytics/src/consistentAnonymousId.ts

```diff
@@ -1,2 +0,0 @@
-import { v4 as uuidv4 } from 'uuid';
```

Comment:
Cleaning up unused... This also would have generated a new id each app start / cli start